templates/coursecard.mustache

```diff
@@ -52,7 +52,7 @@
         {{! Progress. }}
         {{#progressenabled}}
         <div class="dashboard-card-meta mt-auto px-3">
-            <small>Progress: {{progress}}%</small>
+            <small>{{#str}} progress {{/str}}: {{progress}}%</small>
```

Comment:
Suggestion: make "Progress:" a lang string in this plugin since concatenating punctuation like this (i.e. ':' in this case) assumes the language will only ever be LTR.

I don't know if UCL support non-English language packs but avoiding this assumption is good practice. I'm sure I saw this in the docs somewhere -- i.e. that punctuation should be included in the language string for RTL support -- but I can't find this now.

Reply:
UCL don't have a language menu - there is a statement somewhere that english is used as language for online things.